styles/status.cafe/catppuccin.user.css

```diff
@@ -0,0 +1,134 @@
+/* ==UserStyle==
+@name status.cafe Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/status.cafe
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/status.cafe
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/status.cafe/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Astatus.cafe
+@description Soothing pastel theme for status.cafe
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+
+@var checkbox colors "Accent color links" 1
+==/UserStyle== */
+
+@-moz-document regexp("https://status\\.cafe/(?!users).*") {
```

Comment:
Why can't we theme the users page?

Reply:
As stated before, the users page can have custom css and allowing the theme to work there that can interfere with that.
You can see an example on my own [status.cafe profile](https://status.cafe/users/bunfluff_)

Reply:
Ah right sorry!